### PR TITLE
Update Weather-Observation-Station-10.sql

### PR DESCRIPTION
**Pull request by Ibrahim-netizen**
Suited for mysql and Postgres DBs.

### CODE REVIEW

File: Basic Select/Weather-Observation-Station-10.sql [hunk 1]
```diff
@@ -1,8 +1,4 @@
-# Author: Thomas George Thomas
-select distinct(city) from station where
-city not like "%a" and
-city not like "%e" and
-city not like "%i" and
-city not like "%o" and
-city not like "%u";
+# Author: Ibrahim Giwa
```
Comment (Thomas-George-T) [inline Basic Select/Weather-Observation-Station-10.sql:1]: Hi,
Could you not delete the previous code and add your code below it

Comment (Ibrahim-netizen) [reply Basic Select/Weather-Observation-Station-10.sql:1]: Hi, I believe it requires your approval is why I can’t delete the previous code.

Comment (Thomas-George-T) [reply Basic Select/Weather-Observation-Station-10.sql:1]: The approval is for merging. Please could you make a new commit with the changes in your branch. You can also raise a new PR with the new code below the original code in the file

**Comment by Ibrahim-netizen** [comment]
Hi,

I just tried doing that now. Does that work please let me know?

Thank you,

Ibrahim

**Comment by Thomas-George-T** [comment]
I see the new request. I think the issue still persists. Please pull the file again and add your code to the end. So GitHub should recognize the code as adding new code and not deleting the existing old code in the file. A good way to check this would be to see if the file is having only + in green and no - in red

**Pull request closed without merging**